tutorials/adding_visuals.md

```diff
@@ -0,0 +1,118 @@
+\page adding_visuals
+
+# Overview
+
+This tutorial describes how to import 3D meshes into Gazebo to increase the
+visual fidelity of your model. Continuing with our example, we'll attach a 3D
+mesh to our turtle, making it look much better.
+
+## Related tutorials
+
+https://gazebosim.org/api/sim/8/meshtofuel.html
+https://classic.gazebosim.org/tutorials?tut=import_mesh&cat=build_robot
+https://classic.gazebosim.org/tutorials?cat=guided_i&tut=guided_i2
```

Comment:
These are Gazebo classic tutorials, maybe add a note indicating that these tutorials are from an older version and some things might be different or change in future versions

Reply:
Thanks, added. [1913d7d](https://github.com/gazebosim/gz-sim/pull/2257/commits/1913d7dddeabd6f98b603bda896bf67aa0174439)